napytau/gui/components/checkbox_panel.py

```diff
@@ -38,11 +38,13 @@ def update_data_checkboxes_fitting(self) -> None:
 
         # Update all checkboxes for the fitting
         for i in range(len(self.parent.datapoints)):
-            x, y = self.parent.datapoints[i]
+
+            i_shi, i_unshi = self.parent.datapoints[i].get_intensity()
```

Comment:
```suggestion
            shiftend_intensity, unshifted_intensity = self.parent.datapoints[i].get_intensity()
```
Try to avoid overly shortening variable names